src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.InvalidDataException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    public User getDefaultValidUser() {
+        User user = new User();
+        user.setLogin("validLogin");
+        user.setId(38628L);
+        user.setPassword("unuwjf8j2");
+        user.setAge(22);
+        return user;
+    }
+
+    @Test
+    void registrationDataIsValid() throws InvalidDataException {
+        RegistrationService registrationService = new RegistrationServiceImpl();
+        User user = getDefaultValidUser();
+        try {
+            registrationService.register(user);
+        } catch (InvalidDataException e) {
+            throw new InvalidDataException("Oops, your input data is not valid");
+        }
```

Comment:
why are we expecting an exception if this user is valid?
is the valid data test we should assert the result the we get from the `register()` method call and storage actually contains this user now